src/Microsoft.Sbom.Api/Utils/Constants.cs

```diff
@@ -19,7 +20,25 @@ public static class Constants
         Version = "2.2"
     };
 
-    public static SbomSpecification SPDX22Specification = SPDX22ManifestInfo.ToSBOMSpecification();
+    public static ManifestInfo SPDX30ManifestInfo = new ManifestInfo
+    {
+        Name = "SPDX",
+        Version = "3.0"
+    };
+
+    public static SbomSpecification SPDX22Specification = ManifestInfo.ToSBOMSpecification(SPDX22ManifestInfo);
+
+    public static Collection<SbomSpecification> SupportedSbomSpecifications = new()
+    {
+        ManifestInfo.ToSBOMSpecification(SPDX22ManifestInfo),
+        ManifestInfo.ToSBOMSpecification(SPDX30ManifestInfo),
+    };
+
+    public static Collection<ManifestInfo> SupportedSpdxManifests = new()
+    {
+        SPDX22ManifestInfo,
+        SPDX30ManifestInfo,
+    };
 
     // TODO: move to test csproj
```

Comment:
Please open a separate PR to move the `TestManifestInfo` after this PR is merged